### PR TITLE
at, ifdown, ifup, ip-address, ip: add French translation

### PR DESCRIPTION
**Pull request by patricedenis**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by patricedenis** [comment]
@bl-ue : #5501 
> 
@patricedenis only repository collaborators and organization members can add/remove issue/PR labels. Seeing "translation" in the PR title thought makes it very easy to identify it as a translation and so it's not much work to label, but thank you for the offer!
>
OK I did'nt know that :smiley: 
>
I'm not sure if GitHub supports PR templates like it does issue templates. If it did, we could create PR templates for translations, page edits, new commands, etc., and have labels automatically added when a user creates a PR. With issue templates, you can configure which labels should automatically be applied when a user opened an issue thinking
> 

All right, I think then I may need to move to a CLI git client to continue. I'm thinking that it seems really a gain in time to apply those kind of templates. I already use git for my local files and I did know about templates but never put them in use. Have you already written some .gitmessage templates that auto detect the labels that should be used into the commit message?

**Comment by bl-ue** [comment]
Actually, that was just wishfull thinking on my part.  I was refering to a GitHub-only (it's not part of plain Git 😉) feature documented [here](https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates).

File: pages/linux/ifdown.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # ifdown
 
 > Disable network interfaces.
+> More information: <https://linux.die.net/man/8/ifdown>.
```
Comment (sbrl) [inline pages/linux/ifdown.md:4]: I think generally we're moving to man7.org links here, but this is a very interesting case where linux.die.net has the man page, but man7.org doesn't.

`at` and `ip` do (and I see this PR correctly links to those), but `ifup` and `ifdown` don't.

I can't remember why we prefer man7 over linux.die.net though.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

File: pages/linux/ifdown.md (hunk 1 above)
Comment (bl-ue) [reply pages/linux/ifdown.md:4]: #5143. Per @mebeim:

>[man7.org] IMO renders the pages a lot better (sometimes die.net pages have syntax errors due to special characters being treated strangely when rendered in HTML, furthermore man7.org pages are actually in monospace).

Comment (bl-ue) [reply pages/linux/ifdown.md:4]: Let's just leave it for now.  I'll update it later perhaps with the rest of them (if I can find an alternative.)

**Review by bl-ue — APPROVED** [review]

File: pages/linux/ifdown.md (hunk 1 above)
Comment (patricedenis) [reply pages/linux/ifdown.md:4]: @sbrl : yes that's why I put the die.net links here because I could not find them in man7.org ^^

**Pull request merged**